### PR TITLE
#166840908 Adds HoundCI badge to readme

### PR DESCRIPTION
**Pull request by tolumide-ng**
#### What does this PR do?
Adds HoundCI badge

#### Description of Task to be completed?
- Adds HoundCI badge to readme.md
- Repositions parserOptions rule on .eslintrc
- Changes the cmd on package.json to be specific

#### How should this be manually tested?
Clone the repo, cd into it and run **npm run start:dev**

#### What are the relevant pivotal tracker stories?
[#166840908](https://www.pivotaltracker.com/story/show/166840908)

### CODE REVIEW

**Comment by kechyy** [comment]
Good Job Tolumide. Looks good to me.

File: src/helpers/util.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const logger = console;
```
Comment (segunolalive) [inline src/helpers/util.js]: why are you storing the `console` global in a variable?

Comment (tolumide-ng) [reply src/helpers/util.js]: Thank you @segunolalive  for the question. eslint flags using console.log on any file. So, I saved it as variable and utilised it on server file, hence stopping the flag.

Comment (segunolalive) [reply src/helpers/util.js]: If it's been decided to not have `console.log` in the codebase, the question would be why was that decision made? Is it useful? If it is, why try to bypass the tool you use to ensure compliance.

Someone can choose to something like.
`beans = console` and now we'll have `beans` all over the app. Or more realistically, everyone uses this `logger` all over the place. What then would have been the point of adding a `no-console` rule?

Think about this and possible solutions

Comment (tolumide-ng) [reply src/helpers/util.js]: Thank you for this wonderful insight I would remove that now, but would it be okay to change it to [consola](https://www.npmjs.com/package/consola) ?

Comment (segunolalive) [reply src/helpers/util.js]: Some of the options available are node's `process.stdout.write()`, `console.log` or a logger like ~`morgan`~. Morgan is a request logger.

Comment (segunolalive) [reply src/helpers/util.js]: I don't think this justifies adding a new dependency.

Comment (tolumide-ng) [reply src/helpers/util.js]: Thank you for your feedback, I would implement process.stdout.write() now

**Comment by segunolalive** [comment]
Good work @tolumide-ng

**Pull request merged**